sample/src/main/kotlin/com/slack/circuit/sample/petdetail/PetDetail.kt

```diff
@@ -121,35 +124,49 @@ class PetDetailScreenFactory @Inject constructor() : ScreenViewFactory {
   }
 }
 
-private fun petDetailUi() = ui<PetDetailScreen.State, Nothing> { state, _ -> RenderImpl(state) }
+private fun petDetailUi() = ui<PetDetailScreen.State, Nothing> { state, _ -> PetDetail(state) }
 
 @Composable
-private fun RenderImpl(state: PetDetailScreen.State) {
+internal fun PetDetail(state: PetDetailScreen.State) {
   Scaffold(modifier = Modifier.systemBarsPadding()) { padding ->
     when (state) {
       PetDetailScreen.State.Loading -> {
         Box(modifier = Modifier.fillMaxSize(), contentAlignment = Alignment.Center) {
-          CircularProgressIndicator()
+          CircularProgressIndicator(modifier = Modifier.testTag("progress"))
```

Comment:
iirc there's a built-in one to progress indicators we could use. Maybe @seankim-android has suggestions?

Reply:
I did a quick search and found [this](https://medium.com/@adrian.gl/how-to-test-jetpack-composes-progressindicators-64bd231e1b22). It doesn't seem super helpful in this case though as we just need to assert that the progress indicator - which just spins and doesn't indicate any actual progress - is visible.

Was there something in particular you were thinking I could assert?